cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,89 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
+  const testData = {
+    user: {
+      username: 'Adam',
+      password: '12345Qwert!'
+    },
+
+    book:{
+      name: 'Git Pocket Guide',
+      author: 'Richard E. Silverman',
+      publisher: "O'Reilly Media",
+      description: 'This pocket guide is the perfect on-the-job companion to Git, the distributed version control system.'
+    },
+
+    alerts: {
+      bookAdded: 'Book added to your collection.',
+      bookDeleted: 'Book deleted.'
+    }
+  };
+
+  beforeEach(() => {
+    cy.viewport(1080, 1280);
+    cy.visit('https://demoqa.com/login');
+  });
+
+  it('should allow existed user to login', () => {
+    cy.findByPlaceholder('UserName')
+      .type(testData.user.username)
```

Comment:
try to be more consistent in your code. do not mix strings with `;` at the end with other strings without `;`
we recommend using it where needed